src/messages/rdf.js

```diff
@@ -71,6 +71,30 @@ export const RDF_TYPE_MISSING = {
   file: INSTALL_RDF,
 };
 
+export function rdfMultipleTags(tag) {
+  return {
+    code: `RDF_MULTIPLE_${tag.toUpperCase()}_TAGS`,
+    message: _(`Multiple <${tag}> elements found`),
+    description: _(oneLine`There should be only one tag <${tag}>`),
```

Comment:
You don't need the `oneLine` helper here. It's only used to strip newlines and whitespaces if a string spans across multiple lines.